### PR TITLE
Update Mima

### PR DESCRIPTION
**Pull request by zarthross**
and a few other random things needing updated.

### CODE REVIEW

File: .github/workflows/clean.yml [hunk 1]
```diff
@@ -12,11 +12,12 @@ on: push
 jobs:
   delete-artifacts:
     name: Delete Artifacts
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```
Comment (isomarcte) [inline .github/workflows/clean.yml:15]: Why the change to macos?

Comment (zarthross) [reply .github/workflows/clean.yml:15]: Wasn't on purpose, seemed to change when workflow was regenerated.... I'll take a look.

Comment (isomarcte) [reply .github/workflows/clean.yml:15]: nbd, I see both ubuntu and macos in the CI. Bigger deal is CI failure.

Comment (zarthross) [reply .github/workflows/clean.yml]: I corrected it by changing the order of the `githubWorkflowOSes` key.

**Comment by zarthross** [comment]
Looks like the build passes now.  Do you want to major version bump this since its a mima major version bump?
